eng/scripts/Get-WingetVersion.Tests.ps1

```diff
@@ -0,0 +1,18 @@
+Set-StrictMode -Version 4
+
+Describe 'Valid verison numbers' { 
+    It "Given <version> returns <expected>" -ForEach @(
+        @{ version = '0.4.0-beta.2'; expected = '0.4.0.2' },
+        @{ version = '0.4.1'; expected = '0.4.1.0' },
+        @{ version = '0.4.2'; expected = '0.4.2.0' },
+        @{ version = '1.0.0-beta.1'; expected = '1.0.0.1' },
+        @{ version = '1.0.0'; expected = '1.0.0.0' },
```

Comment:
I thought it was precisely for this reason you didn't want to do this scheme and instead use the MSI-based scheme where the PATCH semver field is +1 when GA'd?

Reply:
Thanks for the chat weeks back. We went ahead and used the MSI versioning format for consistency across all the Windows packages and package managers.